docs/source/index.rst

```diff
@@ -0,0 +1,125 @@
+IMAS-Core Documentation
+=======================
+
+.. image:: https://github.com/iterorganization/IMAS-Core/actions/workflows/wheels.yml/badge.svg?branch=develop
+    :target: https://github.com/iterorganization/IMAS-Core/actions/workflows/wheels.yml
+    :alt: Build Status
+
+.. image:: https://img.shields.io/github/license/iterorganization/IMAS-Core
+    :target: https://github.com/iterorganization/IMAS-Core/blob/main/LICENSE.txt
+    :alt: License
+
+
+What is IMAS?
+-------------
+
+**IMAS** (Integrated Modeling and Analysis Suite) is ITER's comprehensive suite for storing and analyzing fusion experiment data.
```

Comment:
I think we have to go beyond the idea that IMAS is just about data storage and access and so would say:
```suggestion
**IMAS** (Integrated Modeling and Analysis Suite) is ITER's comprehensive suite for simulating and analyzing fusion data.
```